WizardExtensions/MSTestv2IntelliTestExtension/MSTestv2TestFramework.cs

```diff
@@ -127,7 +127,7 @@ public override TypeName ExpectedExceptionAttribute
             {
                 if (this.expectedExceptionAttribute == null)
                 {
-                    this.expectedExceptionAttribute = MSTestv2TestFrameworkMetadata.AttributeName("ExpectedExceptionAttribute");
+                    this.expectedExceptionAttribute = MSTestv2TestFrameworkMetadata.AttributeName("ExpectedException");
```

Comment:
[This](https://github.com/Microsoft/testfx/blob/master/WizardExtensions/MSTestv2IntelliTestExtension/MSTestv2TestFrameworkMetadata.cs#L46) AttributeName is now appending the "Attribute" keyword. In MSTestV1, it was not appending the "Attribute" keyword in AttributeName function and we had to append it in arguments itself.